objects/sv-SE.json

```
@@ -1,25 +1,25 @@
 {
   "rct2.glthent": {
     "reference-name": "'Goliath' Sign",
-    "name": "'Goliath' skylt"
+    "name": "’Goliat’-skylt"
```

```suggestion
    "name": "’Goliath’-skylt"
```

Refers to a ride called Goliath, not to the biblical giant.

---

Ok, I see.